packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.vue

```diff
@@ -8,16 +12,17 @@ import {
   type DataGridRendererRow,
   type OnyxDataGridProps,
 } from "../..";
-import { useDataGridFeatures } from "./features";
-import { useDataGridSorting } from "./features/sorting/sorting";
+import { useDataGridFeatures, type DataGridFeature } from "./features";
 
-const props = defineProps<OnyxDataGridProps<TEntry>>();
+const props = withDefaults(defineProps<OnyxDataGridProps<TEntry, TFeatures>>(), {
+  features: () => [] as TFeatures,
+});
 
-const withSorting = useDataGridSorting<TEntry>();
-
-const { watchSources, createRendererRows, createRendererColumns } = useDataGridFeatures([
-  withSorting,
-]);
+// eslint-disable-next-line vue/no-setup-props-reactivity-loss
+const { watchSources, createRendererRows, createRendererColumns } = useDataGridFeatures<
+  TEntry,
+  TFeatures
+>(props.features);
```

Comment:
Shouldn't we use them reactively so the user can add/remove features programtically? E.g. if they have a table configurator for the user where he can enable/disable filtering/sorting or feature flags are used